src/test/functional/queryServices.test/suggested-tags.integration.ts

```diff
@@ -238,6 +239,16 @@ describe('tags query tests - happy path', () => {
           api_id: 'apiid',
           api_id_updated: 'updated_api_id',
         },
+        {
+          user_id: 1,
+          item_id: 10,
+          tag: 'tag B',
+          status: 1,
+          time_added: date4,
+          time_updated: date4,
+          api_id: 'apiid',
+          api_id_updated: 'updated_api_id',
+        },
```

Comment:
this is why the test is failing -- added more data. Probably a leftover from when you were trying to get the suggested tags test to fail?

Reply:
oh yes, thanks for catching that Kat - shifting branches moves the unstaged changes too 🤦